### ISSUE TITLE
Traitor traps that cannot be interacted but only visible

### ISSUE THREAD

**Issue by ThePlatinumGhost** [TTT-2/TTT2#501]
If thats possible, you could add a cvar similar to "ttt_role_traitor_button" which allows the role to see traitor traps but cannot activate them, like "ttt_role_traitor_button_onlysee"
the name of the cvar of your choice of course

**Comment by Alf21** [comment]
That's nothing that would make sense imo. It would make sense to have a hook that could be called to activate the visible effect, e.g. for an innocent role seeing the position of T-traps.

**Comment by ThePlatinumGhost** [comment]
> That's nothing that would make sense imo. It would make sense to have a hook that could be called to activate the visible effect, e.g. for an innocent role seeing the position of T-traps.

yes, in fact maybe I explained myself wrong, I was talking about a similar thing but to give it only to some classes of your choice
for example i would give to clairvoyant 

knowing the locations of the traps / t-rooms can be an advantage
